dask/dataframe/core.py

```diff
@@ -2605,21 +2605,30 @@ def clip_upper(self, threshold):
         return self.map_partitions(M.clip_upper, threshold=threshold)
 
     @derived_from(pd.DataFrame)
-    def squeeze(self, axis=1):
-        if axis == 1:
+    def squeeze(self, axis=None):
+        if axis in [None, 1]:
+            squeezed_series = self[self.columns[0]]
+
             if len(self.columns) == 1:
                 if len(self.index) == 1:
-                    # return scalar if dataframe contains single value to
-                    # match pandas implementation
-                    return self.values.compute()[0]
+                    # dask dataframe with only one element should return scalar
+                    dask_arr = squeezed_series.values
```

Comment:
@mrocklin have attempted to avoid triggering computation whilst still mirroring the `pandas` implementation which returns a scalar in this case, not sure if this is the best way to go about it?